### PR TITLE
slight bug fixes

### PR DESCRIPTION
**Pull request by jenlindner**

### CODE REVIEW

File: app/javascript/App.vue [hunk 1]
```diff
@@ -240,8 +240,8 @@ export default {
       return formData
     },
     addComponents(formData){
-      formData.append(this.etdPrefix('school'), this.form.getSelectedSchool())
-      formData.append(this.etdPrefix('department'), this.form.getSelectedDepartment())
+      formData.append(this.etdPrefix('school'), this.sharedState.getSelectedSchool())
+      formData.append(this.etdPrefix('department'), this.sharedState.getSelectedDepartment())
       formData.append(this.etdPrefix('school'), this.sharedState.getSelectedSchool())
       formData.append(this.etdPrefix('department'), this.sharedState.getSelectedDepartment())
```
Comment (bess) [inline app/javascript/App.vue]: Apologies for my ignorance, but this looks like it's adding the same thing twice. Is that intentional?

Comment (little9) [reply app/javascript/App.vue]: I think this was a merge that I did that went wrong

**Comment by little9** [comment]
👍

**Comment by jenlindner** [comment]
@little9 yep! Here you go, thanks @bess

**Pull request merged**